librz/include/rz_type.h

```diff
@@ -74,6 +74,7 @@ typedef enum {
 	RZ_BASE_TYPE_KIND_ENUM,
 	RZ_BASE_TYPE_KIND_TYPEDEF,
 	RZ_BASE_TYPE_KIND_ATOMIC, // For real atomic base types
+	RZ_BASE_TYPE_KIND_CLASS, // Just a "placeholder" for now.
```

Comment:
I don't think we need to mark it as a placeholder. Some basic features will work already.